src/controllers/seller controller/sellerController.ts

```diff
@@ -113,6 +117,72 @@ class SellerController {
       console.error(error);
     }
   }
+
+  //get a particular item if it is in the collection or list of products belonging to the seller
+  async getSellerProduct(req: Request, res: Response): Promise<void> {
+    const token = req.cookies.jwt;
+    const productId = parseInt(req.params.productId, 10);
+  
+    if (!token) {
+      res.status(401).json({ message: 'Unauthorized: Token is missing' });
+      return;
+    }
```

Comment:
Move this to middleware. Authentication and Authorization should be done by  a middleware